praxis-notes-app/src/stripe/api/get-credit-history.query.ts

```
@@ -0,0 +1,35 @@
+import { Error, Success } from '@errors/utils';
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)